compiler/src/dmd/backend/arm/cod1.d

```diff
@@ -1314,6 +1314,146 @@ void fixresult(ref CodeBuilder cdb, elem* e, regm_t retregs, ref regm_t outretre
     }
 }
 
+/*******************************
+ * Extra information about each CLIB runtime library function.
+ */
+
+private
+enum CLIB
+{
+    realToDouble,
+    doubleToReal,
+}
+
+private
+struct ClibInfo
+{
+    regm_t retregs;     // registers that 32 bit result is returned in
+}
+
+int clib_inited = false;          // true if initialized
```

Comment:
`bool` also `__gshared`?

Reply:
Good you saw that! There are a few things like that I want to improve, but this PR is just a rough in.